index.html

```diff
@@ -0,0 +1,177 @@
+          <p class="p-1">
+            Play snooker with other players over a chessboard, rather than
+            playing online.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-book"></i>
+          </div>
+          <h3 class="program-title">Theory</h3>
+          <p class="p-1">
+            Study the important theory about the openings, middle game and
+            endgame.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-users"></i>
+          </div>
+          <h3 class="program-title">Connect</h3>
+          <p class="p-1">
+            Become part of the amazing snooker community to discuss, study and
+            play.
+          </p>
+        </section>
+      </section>
+      <button type="button" id="join-btn">Join Event</button>
+    </article>
+    <article class="feature-container d-flex flex-column align-items-center p-3 bg-white" id="player-section">
+      <h2 class="section-heading">Featured Players</h2>
+      <hr />
+      <section class="row justify-content-center">
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player1">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player2">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player3">
+   
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player4">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player5">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player6">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player7">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player8">
+          
+        </section>
+      </section>
+    </article>
+    <article class="partner-container1 d-flex flex-column align-items-center p-2" id="partner-section">
+      <h2 class="section-heading">Partners</h2>
+      <hr />
+      <section class="partner-grid d-flex flex-wrap align-items-center p-3 gap-4 justify-content-center">
+        <img src="img/partner1.png" alt="algorand logo" />
+        <img src="img/partner2.png" alt="algorand logo" />
+        <img src="img/partner3.png" alt="algorand logo" />
+        <img src="img/partner4.png" alt="algorand logo" />
+        <img src="img/partner5.png" alt="algorand logo" />
+      </section>
+    </article>
+  </main>
+  <section class="py-3 mx-2 border border-3 text-center" id="more">
+    <strong>MORE</strong> <i class="fa-solid fa-chevron-down mx-2 arrow-down"></i>
+  </section>
+  <footer
```

Comment:
- [ ] The footer contains a logo aligned to the left and text aligned to the right. Kindly apply this to your design
- [ ] And this applies to the mobile version as well

Your design
![image](https://user-images.githubusercontent.com/81932478/204014553-bc7cdc9f-e8ce-4548-9ed5-b348c694c206.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/204014593-8bf765d4-8a53-4cf1-9045-3cd75e9858b8.png)

#### This applies to the about page as well